hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -450,10 +451,7 @@ private void validateLedgerState() {
      */
     @Override
     public void run() {
-        // Start the gRPC servers.
-        logger.info("Starting mono gRPC server");
-        daggerApp.grpcStarter().startIfAppropriate();
-
+        // Start the gRPC server.
```

Comment:
We're going all-in on the new workflows. No more mixed mode for the grpc servers.